lifecycle/src/lifecycle_service_client_py.py

```diff
@@ -24,7 +24,12 @@
 def change_state(lifecycle_node, change_state_args=''):
     node = rclpy.create_node('lc_client_py')
 
-    cli = node.create_client(ChangeState, lifecycle_node + '/change_state')
+    service_name = lifecycle_node + '/change_state'
+    cli = node.create_client(ChangeState, service_name)
+    if not cli.wait_for_service(timeout_sec=5.0):
+        print('Unable to call service', service_name)
```

Comment:
while this work, we usually use either `%s` or `.format()` to format our printed strings.
Also this entire demo now uses ROS logging so I'd suggest to use logging rather than `print`